package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-dashboard-framework",
-  "version": "0.10.1.1",
```

Comment:
I thought you had to make a similar change in some bower file?

Reply:
Yes @david-oc-miller you are right...